### DISCUSSION TITLE
[Projects Beta] - Navigation

### DISCUSSION THREAD

**Discussion by theilgaz** [community/community#7530] [Projects and Issues]
- [ ] Adding an organization header would make it easy to navigate back. There is no direct link to navigate to somewhere else.  
- [ ] By adding a project link to the repository, the projects page can be more easily navigated.

**Comment by pattersongarth** [comment]
This is desperately needed. Navigation is terrible as it is right now.

**Comment by mkungla** [comment]
Collected all related discussions here: https://github.com/github/feedback/discussions/10448
